src/main.cpp

```diff
@@ -3205,7 +3205,7 @@ bool CBlock::ConnectBlock(CTxDB& txdb, CBlockIndex* pindex, bool fJustCheck, boo
         }
 
         //Approve first coinstake in DPOR block
-        if (IsResearcher(bb.cpid) && IsLockTimeWithinMinutes(GetBlockTime(),15) && !IsResearchAgeEnabled(pindex->nHeight))
+        if (IsResearcher(bb.cpid) && IsLockTimeWithinMinutes(GetBlockTime(), GetAdjustedTime(), 15) && !IsResearchAgeEnabled(pindex->nHeight))
```

Comment:
This only affects blocks before ResearchAge. Which was long time ago. looks like api change

Reply:
`IsLockTimeWithinMinutes` used to use `GetAdjustedTime()` so the result here will be identical. We can probably eliminate a lot of these type of checks due to being bypassed by grandfather block anyway.